include/rmm/aligned.hpp

```diff
@@ -43,9 +43,9 @@ static constexpr std::size_t CUDA_ALLOCATION_ALIGNMENT{256};
 /**
  * @brief Returns whether or not `value` is a power of 2.
  *
- * @param[in] value to check.
+ * @param[in] value value to check.
  *
- * @return Whether the input a power of two with non-negative exponent
+ * @return True if the input a power of two with non-negative exponent, false otherwise.
```

Comment:
```suggestion
 * @return True if the input is a power of two with non-negative exponent, false otherwise.
```

Reply:
Double-nit (no need to act on it) `non-negative integer exponent` (all integers can be expressed as powers of two if we admit real exponents).